### PR TITLE
docs: add roadmap

### PR DESCRIPTION
**Pull request by Leeway213**

### CODE REVIEW

**Review by ShenhaoZhu — APPROVED** [review]

**Review by Leoooo333 — APPROVED** [review]

File: docs/ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+## Champ
+
+#### Milestone Summary
+
+| Status | Milestone                                                                                  | Goals |       ETA       |
+| :----: | :----------------------------------------------------------------------------------------- | :---: | :-------------: |
+|   ✅   | **[Model and test data on Huggingface](https://huggingface.co/fudan-generative-ai/champ)** | 2 / 2 | Tue Mar 26 2024 |
+|   🚀   | **[Data preprocessing code release]()**                                                    | 0 / 1 | Thu Mar 28 2024 |
+|   🚀   | **[Trainging code release]()**                                                             | 0 / 1 | Mon Apr 01 2024 |
```
Comment (suqingkun) [inline docs/ROADMAP.md:15]: Add the milestone of releasing training dataset

**Review by suqingkun — COMMENTED** [review]
Add the milestone of releasing training dataset. The date is same with training code.

File: docs/ROADMAP.md (hunk 1 above)
Comment (Leeway213) [reply docs/ROADMAP.md]: done

**Review by suqingkun — APPROVED** [review]

**Comment by Leeway213** [comment]
docs: add roadmap

**Pull request merged**